multisig/proposals/arbitrumsepolia/vip-007/index.ts

```diff
@@ -0,0 +1,143 @@
+      params: [
+        PRIME_LIQUIDITY_PROVIDER,
+        "setTokensDistributionSpeed(address[],uint256[])",
+        arbitrumsepolia.NORMAL_TIMELOCK,
+      ],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [
+        PRIME_LIQUIDITY_PROVIDER,
+        "setMaxTokensDistributionSpeed(address[],uint256[])",
+        arbitrumsepolia.NORMAL_TIMELOCK,
+      ],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME_LIQUIDITY_PROVIDER, "setMaxLoopsLimit(uint256)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME_LIQUIDITY_PROVIDER, "pauseFundsTransfer()", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME_LIQUIDITY_PROVIDER, "resumeFundsTransfer()", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "updateAlpha(uint128,uint128)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "updateMultipliers(address,uint256,uint256)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "setStakedAt(address[],uint256[])", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "addMarket(address,address,uint256,uint256)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "setLimit(uint256,uint256)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "setMaxLoopsLimit(uint256)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "issue(bool,address[])", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "burn(address)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "togglePause()", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: PRIME_LIQUIDITY_PROVIDER,
+      signature: "setPrimeToken(address)",
+      params: [PRIME],
+    },
+    {
+      target: COMPTROLLER_CORE,
+      signature: "setPrimeToken(address)",
+      params: [PRIME],
+    },
```

Comment:
I think we don't need this command yet. We'll execute in the VIP setting the Prime markets. I suppose markets from the Core pool comptroller will be added, but potentially only markets from other pools could be added, and in that case we wouldn't need the Prime contract set on the Core pool comptroller

Reply:
Got it. https://github.com/VenusProtocol/vips/pull/264/commits/ca43ab0b478ce3d97555ab0b0af5f0b2e36e3122